### PR TITLE
#318 - 'Time Passed' should show `<1min` instead of seconds and update constantly.

### PR DESCRIPTION
**Pull request by giladt**
## What was done
`Last activity` shows '<1min' if less than a minute has passed since the last activity has happened.
Applies to `discussions list` activity column and `single comment` time since created.

Preview:
https://prime-deals-dapp-git-fix-discussions-list-shou-3dd68b-curvelabs.vercel.app/deal/open_deals_stream_hash_2

### CODE REVIEW

File: src/dealDashboard/discussionsService.ts [hunk 1]
```diff
@@ -160,6 +170,7 @@ export class DiscussionsService {
         createdBy,
         createdAt: new Date(),
         modifiedAt: new Date(),
+        lastModified: "<1min",
```
Comment (dkent600) [inline src/dealDashboard/discussionsService.ts:173]: Can please be "< 1 min"?

File: src/services/DateService.ts [hunk 2]
```diff
@@ -45,8 +45,8 @@ export class DateService {
       relativeTime: {
         future: "in %s",
         past:   "%s ago",
-        s:  "%dsec",
-        ss: "%dsec",
+        s:  "<1min",
```
Comment (dkent600) [inline src/services/DateService.ts:48]: Please add spaces to all of these between the number and the unit.  "m" should be "mo"

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/services/DateService.ts (hunk 2 above)
Comment (giladt) [reply src/services/DateService.ts:48]: done

File: src/dealDashboard/discussionsService.ts (hunk 1 above)
Comment (giladt) [reply src/dealDashboard/discussionsService.ts:173]: done

**Review by dkent600 — APPROVED** [review]

**Pull request merged**